packages/core/content-manager/admin/src/hooks/useDocumentActions.ts

```diff
@@ -29,21 +31,24 @@ import type {
   Clone,
   Create,
   Delete,
+  BulkDelete,
   Discard,
   FindOne,
   Publish,
   Update,
   Unpublish,
+  BulkUnpublish,
 } from '../../../shared/contracts/collection-types';
 
 const DEFAULT_UNEXPECTED_ERROR_MSG = {
   id: 'notification.error',
   defaultMessage: 'An error occurred, please try again',
 } satisfies MessageDescriptor;
 
-type OperationResponse<TResponse extends { data: any; meta: any; error?: any }> =
+type OperationResponse<TResponse extends { data: any; meta?: any; error?: any }> =
   | Pick<TResponse, 'data' | 'meta'>
-  | { error: BaseQueryError | SerializedError };
+  | { error: BaseQueryError | SerializedError }
+  | Pick<TResponse, 'data'>;
```

Comment:
can we add comments in what circumstances we'd get this over the first? if it's for the many operations, maybe we should have a completely different type so we dont have to constantly check if `meta` is defined etc.

Reply:
yes its for "many"